tutorials/colors/colormap-manipulation.py

```diff
@@ -255,6 +255,50 @@ def plot_linearmap(cdict):
 
 plot_examples([cmap1, cmap2])
 
+#############################################################################
+# .. _reversing-colormap:
+#
+# Reversing a colormap
+# ====================
+#
+# `.Colormap.reversed` creates a new colormap that is a reversed version of
+# the original colormap.
+
+colors = ["#ffffcc", "#a1dab4", "#41b6c4", "#2c7fb8", "#253494"]
+my_cmap = ListedColormap(colors, name="my_cmap")
+
+my_cmap_r = my_cmap.reversed()
+
+plot_examples([my_cmap, my_cmap_r])
+# %%
+# This method reverses the colormap and, if no name is passed in, names the
+# copy by appending ``_r`` to the original colormap's name.
+# See :ref:`registering-colormap` for an example.
```

Comment:
```suggestion
# See the following section for an example.
```

Small nit, you can take it or leave it: Linking to just the line below feels rather distracting.